src/Microsoft.DotNet.VersionTools/Dependencies/MSBuildPackageReferenceUpdater.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.VersionTools.Util;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.VersionTools.Dependencies
+{
+    public class MSBuildPackageReferenceUpdater : IDependencyUpdater
+    {
+        public IEnumerable<string> MSBuildFilePaths { get; }
+
+        public MSBuildPackageReferenceUpdater(IEnumerable<string> msbuildFilePaths)
```

Comment:
One thing we would lose if we removed this would be the validation part, which could kick in if someone accidentally hard-coded a version string into a csproj file that later fell out of sync. If we kept this, we'd still get that validation, even if we didn't plan to use it for actually updating version strings directly.